### DISCUSSION TITLE
[IMPORTANT] For those who Bookmarklets dont work, heres why.

### DISCUSSION THREAD

**Discussion by luh-99** [luh-99/School-Stuff#4] [Announcements]
### First, what are Bookmarklets?
Bookmarklets are bookmarks that contain javascript to update a website/webpage, which is a little more easier than executing them in the javascript console with the inspect element. In our case, we want to use bookmarklets to execute certain javascript commands for exploits such as LTBEEF, to open up things like an UI Menu. This is perfectly normal for this case.

### Ok, then why don't they work?
Ok, so lets say you were trying to perform LTBEEF one day, except, the menu doesn't load, and your like "Hmm, did my school block bookmarklets?" Well, its here's how you can check:

1. First, go to `chrome://policy` to see what your school admins are adding as restrictions.
2. Then you want to scroll down to "url blocklist" or search it up with ctrl-f, and it should look something like this
![280294687-405e235d-919e-4520-bc72-38c3faeef998](https://github.com/user-attachments/assets/8a00a324-353d-4ce3-979f-07c3b735f6b6)
 and if it says `"javascript://"` in mono space next to url blocklist, then your school has blocked Bookmarklets :/. You can check what else your school blocks here as well.

### Ok, now what?
So, currently, there isn't any exploits to re-enable bookmarklets besides unenrollment, but at that point what are exploits for? And using the linux terminal is most likely blocked, so for now, you just have to wait for people like me or mercury workshop to find out how, or try scrolling through bypassi.
If you have any other questions, just ask below, because I was just trying to keep this super simple.

**Comment by Braden000** [comment]
chrome://crostini-installer/ for linux  but my chromebook doesn't have enough space

**Reply by shragsir** [reply]
?

More details?

**Reply by shragsir** [reply]
I will try to help you best I can after more info.
